src/app/admin/edit-product-form/index.tsx

```diff
@@ -144,8 +156,8 @@ export default function EditProductForm({ product, getProducts, onClose }: Props
                 {...form.getInputProps('available')}
               />
               <RoundButton
-                text={editing ? 'Update' : 'Submit'}
-                onClick={() => (editing ? console.log('Editing') : console.log('Creating'))}
+                text={product ? 'Update' : 'Submit'}
+                onClick={() => (product ? console.log('Editing') : console.log('Creating'))}
```

Comment:
onClick is necessary since it's defined in the component ()
`interface Props {text: string, onClick: () => void}`

`export function RoundButton({ text, onClick }: Props)`